Move parameters out to real bundle configuration

rmasters opened the pull request.

rmasters committed:
Move parameters out to real bundle configuration

ollietb commented on Resources/config/config.yml:
```diff
@@ -0,0 +1,16 @@
+            oauth_path: /oauth/authorize/?client_id={client_id}&amp;response_type=code&amp;redirect_uri={redirect_uri}
+            oauth_token_path: /oauth/access_token/
+            redirect_route: OhInstagramBundle_callback
+            no_verify_peer: true
```
@rmasters can you change the default to false here so it's more secure?

rmasters replied:
@ollietb Actually, that file isn't even loaded - I'll remove it.

rmasters committed:
Removed unused config.yml

See DependencyInjection/Configuration.php

rmasters committed:
Make sure if there is no user-config for the bundle, it still works

The user will get an error about an invalid client id, rather than a
key-not-found error from OhInstagramExtension

alex-lancer commented:
Hi! After composer update command I have got an exception:

  [Instaphp\Exceptions\InstaphpException]  
  Invalid client id         

Maybe it needs some update in configuration?

rmasters commented:
@alex-lancer Hi. Did you update your configuration as below? I still need to update the README to document this:

``` yaml
oh_instagram:
    instaphp:
        config:
            client_id: %instagram_api_client_id%
            client_secret: %instagram_api_client_secret%
```

rmasters commented:
@ollietb We should probably tag 31da669 as 1.0, and b1ab0cd as 2.0 since this is a bc-break

alex-lancer commented:
Thank you for your answer! When I read documentation there was nothing about this configuration. I have done what you have described before and it works.
